src/frontend/screens/Game/GamePage/index.css

```diff
@@ -505,8 +505,12 @@
         color: var(--navbar-active);
       }
 
+      &:first-child {
+        margin-top: -0.1rem;
```

Comment:
let's do `0` here instead of a negative margin, I'd rather avoid negative margins as much as possible